main.go

```diff
@@ -386,7 +386,7 @@ func (o *Options) runServerMode() error {
 
 	// force a data refresh in the background. This is important to initially populate the db's materialized views
 	// if this is the first time starting sippy.
-	go server.RefreshData()
+	go server.RefreshData(true)
```

Comment:
Just a thought that if the only time we need to refreshMetrics prior to refreshing data is on startup to avoid the scrape gap, we could do it before the RefreshData call and that way we don't do it everytime in the RefreshData call.  Also if it is only needed for dbOnlyMode (non dbOnlyMode won't have data presumably) then it could also be an explicit check and call to RefreshMetricsDB